.github/workflows/tebako-pack.yml

```diff
@@ -0,0 +1,45 @@
+name: tebako-pack
+
+on:
+  push:
+    tags: [ 'v*' ]
+  workflow_dispatch:
+  pull_request: # to be removed
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.job }}-${{ github.head_ref || github.ref_name }}'
+  cancel-in-progress: true
+
+env:
+  RUBY_VER: 3.2.3
+
+jobs:
+  pack:
+    name: Pack fontist
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, macos-11.0]
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 1
+
+    - name: Setup Tebako
+      uses: ./.github/actions/setup-tebako
+      with:
+        cache: build
+        ruby_version: ${{ env.RUBY_VER }}
+        bundler_version: 2.3.22
```

Comment:
I believe it is necessary to set bundler version only for Ruby 3.0.x, 2.7.x
3.1.x, 3.2.x shall worl with defaults.  Can we try to use default here ?

Reply:
Removed